pkg/gofr/datasource/sql/sql.go

```diff
@@ -72,6 +74,14 @@ func NewSQL(configs config.Config, logger datasource.Logger, metrics Metrics) *D
 		return database
 	}
 
+	// We are not setting idle connection timeout because we are checking for connection
```

Comment:
I might be tired but I don't get the negation "we are not", while the next line of code apparently does it.

Same remark for the other comment for the other variation sets in this PR

Reply:
we are not setting the timeout we are setting the max idle connection

Reply:
ok, clearer. Thanks. I'm tired then.